package.json

```diff
@@ -18,6 +18,7 @@
     "test": "jest",
     "test:dev": "jest --watch",
     "lint": "eslint src",
+    "prepublishOnly": "yarn test && yarn build",
```

Comment:
```suggestion
    "prepublishOnly": "yarn lint && yarn test && yarn build",
```

Reply:
I didn't add the linting on purpose, because linting currently breaks and I didn't intend to change anything in the code